votes/0186-CabinfeverB-as-pd-committer.md

```diff
@@ -0,0 +1,30 @@
+# A Vote for CabinfeverB as PD Committer
+
+## Proposal
+
+[@CabinfeverB](https://github.com/CabinfeverB) has been working on `tikv/pd` for more than a year. The main contributions are listed in the following:
+
+[scheduling diagnosis](https://github.com/tikv/pd/issues/5257)
+
+[http audit](https://github.com/tikv/pd/issues/3111)
+
+See more:
+* [Authored pull requests](https://github.com/tikv/pd/pulls?q=is%3Apr+author%3ACabinfeverB+is)
+
+I ([@lhy1024](https://github.com/lhy1024)) hereby nominate [@CabinfeverB](https://github.com/CabinfeverB) as PD Committer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 72 hours unless there is an objection or not enough votes.
+
+## Result
```

Comment:
The result should be updated after the voting is finished.

Reply:
done